Loop/Managers/LoopAlertsManager.swift

```diff
@@ -0,0 +1,67 @@
+//
+//  LoopAlertsManager.swift
+//  Loop
+//
+//  Created by Rick Pasetto on 6/8/20.
+//  Copyright © 2020 LoopKit Authors. All rights reserved.
+//
+
+import CoreBluetooth
+import LoopKit
+
+/// Class responsible for monitoring "system level" operations and alerting the user to any anomalous situations (e.g. bluetooth off)
+class LoopAlertsManager: NSObject {
+    private var bluetoothCentralManager: CBCentralManager!
+    private lazy var log = DiagnosticLog(category: String(describing: LoopAlertsManager.self))
+    private weak var alertManager: AlertManager?
+    private let bluetoothPoweredOffIdentifier = Alert.Identifier(managerIdentifier: "Loop", alertIdentifier: "bluetoothPoweredOff")
+
+    init(deviceAlertManager: AlertManager) {
+        super.init()
+        bluetoothCentralManager = CBCentralManager(delegate: self, queue: nil)
+        self.alertManager = deviceAlertManager
+    }
+}
+
+// MARK: CBCentralManagerDelegate implementation
+
+extension LoopAlertsManager: CBCentralManagerDelegate {
+    
+    func centralManagerDidUpdateState(_ central: CBCentralManager) {
+        switch central.state {
+        case .unauthorized:
+            switch central.authorization {
```

Comment:
[nit] If there are all `break` why even have this inner switch?

Reply:
[nit] Also, having `break` on the same line as `case` is not Loop standard (even though I know you prefer it).

Reply:
The real reason I left these in here is in case we also want alerts for when BLE is switched in any way that might hinder Looping.  I suppose they aren't really doing any good, other than being able to see the possible states.  I'll remove them and just put a `default` case.